File: src/main/java/no/nav/domain/pensjon/regler/vedtak/MedlemskapForUTEtterTrygdeavtaler.java
package no.nav.domain.pensjon.regler.vedtak;

import no.nav.domain.pensjon.regler.grunnlag.OppfyltVedSammenlegging;

/**
 * @author Magnus Bakken (Accenture) PK-9695, PKPYTON-923
 */
public class MedlemskapForUTEtterTrygdeavtaler extends AbstraktVilkar{



    /**
     * Inneholder informasjon om bruker har inngang gjennom sammenlegging av trygdetid i avtaleland og Norge.
     * Registreres manuelt av saksbehandler.
     */
    private OppfyltVedSammenlegging oppfyltVedSammenlegging;

    /**
     * Inneholder informasjon om bruker har inngang gjennom sammenlegging av trygdetid i avtaleland og Norge på minst 5 år.
     * Registreres manuelt av saksbehandler.
     */
    private OppfyltVedSammenlegging oppfyltVedSammenleggingFemAr;

}

File: src/main/java/no/nav/pensjon/regler/domain/vedtak/MedlemskapForUTEtterTrygdeavtaler.kt
package no.nav.pensjon.regler.domain.vedtak

import no.nav.pensjon.regler.domain.grunnlag.OppfyltVedSammenlegging

class MedlemskapForUTEtterTrygdeavtaler : AbstraktVilkar() {
    /**
     * Inneholder informasjon om bruker har inngang gjennom sammenlegging av trygdetid i avtaleland og Norge.
     * Registreres manuelt av saksbehandler.
     */
    var oppfyltVedSammenlegging: OppfyltVedSammenlegging? = null

    /**
     * Inneholder informasjon om bruker har inngang gjennom sammenlegging av trygdetid i avtaleland og Norge på minst 5 år.
     * Registreres manuelt av saksbehandler.
     */
    var oppfyltVedSammenleggingFemAr: OppfyltVedSammenlegging? = null

}